pom.xml

```diff
@@ -12,7 +12,7 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <testng.version>7.4.0</testng.version>
-        <selenium.java.version>3.141.59</selenium.java.version>
+        <selenium.java.version>4.0.0</selenium.java.version>
```

Comment:
why did you update selenium, did you use something new from the new version?

Reply:
Because the latest version of Selenide (6.0.3) doesn't work with Selenium 3